containers/Dockerfile.buildenv

```diff
@@ -110,12 +220,31 @@ RUN apt-get update -y \
 # - We use python-is-python3 because on Ubuntu 20.04 /usr/bin/python was removed
 #   to aid in tracking down anything that depends on python 2. However, many
 #   scripts use `/usr/bin/env python` to find python so restore it.
+# - We install h5py from apt since that's much faster when cross compiling than
```

Comment:
do you mean "when building the old Ubuntu 18.04 container"? This isn't related to cross-compiling, right? Is the issue that for the old ubuntu no precompiled h5py is available from pip, so it tries to build wheels?